index.html

```diff
@@ -6,15 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="app.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
-    <title>Home page</title>
+    <title>Personal Portfolio</title>
```

Comment:
- [OPTIONAL] - Please re-write the title tag as the requirement say see the screenshot 👇

![kkk](https://user-images.githubusercontent.com/61562626/130761856-609ce6b7-353a-4fc3-827f-9ca3cbf5424d.jpeg)